.github/workflows/fetch-swaggers.yml

```diff
@@ -0,0 +1,35 @@
+name: Fetch Swaggers
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 0 * * *
+
+jobs:
+  fetch-swaggers:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checking out code
+        uses: actions/checkout@v2
+
+      - name: extract git commit data
+        uses: rlespinasse/git-commit-data-action@v1.x
+
+      - name: Fetch Software Swagger
+        run: wget -O swagger.v3.json https://developer.atlassian.com/cloud/jira/software/swagger.v3.json
+
+      - name: Fetch Platform Swagger
+        run: wget -O swagger-v3.v3.json https://developer.atlassian.com/cloud/jira/platform/swagger-v3.v3.json
+
+      - name: Raise PR
```

Comment:
does this step know to open a PR only if there was a change?

Reply:
Yep